xml/operations-compute-using_flavor_metadata.xml

```diff
@@ -8,143 +8,29 @@
 <section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="topic_vhs_12v_vw">
  <title>Using Flavor Metadata to Specify CPU Model</title>
  <para>
-  In &kw-hos; Nova, the <literal>ComputeCapabilitiesFilter</literal>
+  In &kw-hos; &o_comp;, the <literal>ComputeCapabilitiesFilter</literal>
   scheduling filter matches the exact CPU model of the compute host and the
-  requested compute instance model. It doesn't entertain the libvirt's
-  capability of emulating the hosts CPU model to its guests if the guest's CPU
-  model is older or equal to that of the hosts.
+  requested compute instance model. It does not support the libvirt capability
+  of emulating the host CPU model to its guests if the guest's CPU model is
+  older or equal to that of the hosts.
```

Comment:
Mmh, this final sentence is weird and hard to understand. Does the following replacement make sense? `While libvirt supports emulating guest CPUs that are different from the host CPU, &o_comp; does not support that feature.`

Reply:
tweaked that section. We are gonna have work to do...there are quite a few sentences that are weird and hard to understand. I've been doing my best to ignore them in favor of fixing Very Wrong Stuff.